File: src/senthil/model/cell_voltage.jl
include("solid_potential.jl")

# Cell voltage
# Refer equation 158.

"""Cell voltage across the battery"""
function V(I, c₁::SolidConcentration, c₁ᵣ::SolidRadialConcentration, c₂ᵢₖ::InterfacialConc, q₂ᵢₖ::InterfacialFlux)
    ϕ₁(c₁, c₁ᵣ, I, L, c₂ᵢₖ, q₂ᵢₖ) - ϕ₁(c₁, c₁ᵣ, I, 0, c₂ᵢₖ, q₂ᵢₖ)
end

File: scripts/new_ashwini/model/cell_voltage.m
% Include solid_potential.m file
% solid_potential

% Cell voltage across the battery
function V = cell_voltage(I, c1, c1r, paramsn, paramsp, dn1, dp1)
    V = solid_potential(L, I, c1, c1r, paramsp, dn1) - solid_potential(0, I, c1, c1r, paramsn, dp1);
end
